### PR TITLE
Development

### PR DESCRIPTION
**Pull request by emmanuelkamala**

### CODE REVIEW

**Review by mcrd25 — APPROVED** [review]
# Congratulations! Project Implementation Successful! You may 🚀 

Hi @emmanuelkamala  and @Abdusaid10,

Congratulations you may merge to master! You have successfully implemented your bubble sort methods and they past the code review tests. 🎉 🎊 

## Project Specifications Checklist

- [x] bubble_sort method is implemented and working
- [x] bubble_sort_by method is implemented and working 

You may now move unto Enumerable Methods Project and merge to master! 😄  👍 

***If you have any questions, now or in the future, do not hesitate to contact me on social media.***
**--Maya** (https://twitter.com/mcrd25)
[GitHub (https://github.com/mcrd25)] [Twitter (https://twitter.com/mcrd25)] [linkedin (https://www.linkedin.com/in/mayadouglas/)]

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+def bubble_sort(array)
+    swap = true
+    while swap do
+        swap = false
```
Comment (mcrd25) [inline bubble_sort.rb:4]: Tip for Future: Convention in Ruby is to use 2 space indentation!

Comment (mcrd25) [reply bubble_sort.rb:4]: Seems you are using 4

Comment (emmanuelkamala) [reply bubble_sort.rb:4]: Okay, Noted. 
Thank you.

**Pull request merged**